kedro-airflow/kedro_airflow/plugin.py

```diff
@@ -218,6 +226,8 @@ def create(  # noqa: PLR0913, PLR0912
             # topological sort order obtained from pipeline.nodes, see group_memory_nodes()
             # implementation
             nodes, dependencies = group_memory_nodes(context.catalog, pipeline)
+        elif group_by_namespace:
+            nodes, dependencies = group_by_namespace(pipeline)
```

Comment:
I get this error: `TypeError: 'bool' object is not callable`
I think it's because the `group_by_namespace` flag (which is boolean) and the function `group_by_namespace()` are called the same thing. Maybe it can be called `group_namespace_nodes()` similar to `group_memory_nodes()` it's also not been imported above

Reply:
Ah good call, I was trying to solve that.